### PR TITLE
Update values.py - 2021 Hyundai Sonata

### PR DESCRIPTION
**Pull request by cto424**

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Please include the dongleID for the device these firmware versions were taken from.

**Comment by cto424** [comment]
Hi,

My dongle ID is - 0df9160e1d047a0e

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -214,6 +214,7 @@ class Buttons:
   },
   CAR.SONATA: {
     (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\000DN8_ SCC F-CUP      1.00 1.00 99110-L0000         ',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:217]:
```suggestion
```
duplicate

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -225,6 +226,7 @@ class Buttons:
       b'\xf1\x8799110L0000\xf1\x00DN8_ SCC FHCUP      1.00 1.00 99110-L0000         ',
     ],
     (Ecu.esp, 0x7d1, None): [
+      b'\xf1\000DN ESC \a 106 \a\001 58910-L0100',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:229]:
```suggestion
```

duplicate

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -249,6 +252,7 @@ class Buttons:
       b'HM6M1_0a0_G20',
     ],
     (Ecu.eps, 0x7d4, None): [
+      b'\xf1\x8756310L0010\000\xf1\000DN8 MDPS C 1.00 1.01 56310L0010\000 4DNAC101\xf1\xa01.01',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:255]:
```suggestion
      b'\xf1\x8756310L0010\000\xf1\000DN8 MDPS C 1.00 1.01 56310L0010\000 4DNAC101',
```

New format

File: selfdrive/car/hyundai/values.py [hunk 4]
```diff
@@ -262,12 +266,14 @@ class Buttons:
       b'\xf1\x8756310L0210\x00\xf1\x00DN8 MDPS C 1.00 1.01 56310L0210\x00 4DNAC101',
     ],
     (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\000DN8 MFC  AT USA LHD 1.00 1.01 99211-L0000 191016',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:269]:
```suggestion
```

duplicate

File: selfdrive/car/hyundai/values.py [hunk 5]
```diff
@@ -262,12 +266,14 @@ class Buttons:
...
       b'\xf1\x00DN8 MFC  AT KOR LHD 1.00 1.02 99211-L1000 190422',
       b'\xf1\x00DN8 MFC  AT RUS LHD 1.00 1.03 99211-L1000 190705',
       b'\xf1\x00DN8 MFC  AT USA LHD 1.00 1.00 99211-L0000 190716',
       b'\xf1\x00DN8 MFC  AT USA LHD 1.00 1.01 99211-L0000 191016',
     ],
     (Ecu.transmission, 0x7e1, None): [
+      b'\xf1\x87SALDBA6347404GJ3wwwwff\x86hx\x88\x97\x88\x88\x88\x88\x88vfgf\x88?\xfc\xff\x86Uo\xff\xec/\xf1\x89HT6WA910A1\xf1\x82SDN8G25NB1\000\000\000\000\000\000',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:276]:
```suggestion
```
Duplicate

File: selfdrive/car/hyundai/values.py [hunk 6]
```diff
@@ -249,6 +250,7 @@ class Buttons:
       b'HM6M1_0a0_G20',
     ],
     (Ecu.eps, 0x7d4, None): [
+      b'\xf1\x8756310L0010\000\xf1\000DN8 MDPS C 1.00 1.01 56310L0010\000 4DNAC101',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:253]:
```suggestion
```

**Comment by cto424** [comment]
I performed a factory reset and reinstalled Openpilot.  My Comma2 is not functioning properly now.  It boots up but does not go into the camera view and it doesn't take over when engaging cruise control.  

Could you assist?

**Pull request merged**